app/services/search_field_service.rb

```diff
@@ -61,7 +61,11 @@ class SearchFieldService
     'host_tesim',
     'musician_tesim',
     'printer_tesim',
-    'researcher_tesim'
+    'researcher_tesim',
+    'format_book_tesim',
+    'note_admin_tesim',
```

Comment:
remove note_admin_tesim, as per the ticket, this should not be searchable in Ursus

Reply:
removed